components/accounts/AddressParser.vue

```diff
@@ -14,24 +14,14 @@
   </div>
 </template>
 
-<script lang="ts">
-import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+<script setup lang="ts">
 import { parseBatchAddresses } from './utils'
 import { NeoField, NeoInput } from '@kodadot1/brick'
 
-@Component({
-  components: {
-    NeoField,
-    NeoInput,
-  },
-})
-export default class AddressParser extends Vue {
-  @Prop({ type: Array, default: () => [], required: true }) value!: string[]
-
-  @Emit('input')
```

Comment:
The emitting of the input event is missing in the refactored code.

Reply:
@Jarsen136 thanks for pointing it out. It's now included.